lib/Doctrine/ruleset.xml

```diff
@@ -42,6 +42,8 @@
     <rule ref="Generic.CodeAnalysis.UselessOverridingMethod"/>
     <!-- Forbid inline HTML in PHP code -->
     <rule ref="Generic.Files.InlineHTML"/>
+    <!-- Only one structure (class/interface/trait) should be allowed per file -->
+    <rule ref="Generic.Files.OneObjectStructurePerFile" />
```

Comment:
Isn't that part of some other rule already? Can you add some tests to check it, please?